Cargo.lock

```diff
@@ -107,7 +107,7 @@ dependencies = [
  "getrandom 0.2.15",
  "once_cell",
  "version_check",
- "zerocopy",
+ "zerocopy 0.7.35",
```

Comment:
Did you run `cargo update` on this branch?
IMO all those changes to this file need to be reverted, except the ones that update anemo.